floodwatch/src/js/api/api.js

```diff
@@ -94,7 +94,10 @@ export class APIClient {
 
   async postJSON(path: string, body?: Object): Promise<any> {
     const res = await this.post(path, JSON.stringify(body));
-    return res.json();
+    // if (res.status == 204) {
```

Comment:
blorf, took that out while i was trying to track down a bug and forgot to put it back